github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+(brief description / PR title)
+
+Status: **Ready for Review** or **Open for Visibility**
```

Comment:
There's already a PR template that we've been using in Product for a while, which might also have some relevant sections:
```
(brief description / PR title)

Status: **Opened for visibility**

Reviewers: **(GitHub @<name>s of all reviewers)**
JIRA: **(link to JIRA ticket)**
Linked PRs: **(links to corresponding PRs, optional)**

## Changes
- (change1)

## How to test-drive this PR
- (necessary config changes)
- (necessary corresponding PRs)
- (how to access the new / changed functionality -- fixtures, URLs)

## Research resources
- (link1)
```